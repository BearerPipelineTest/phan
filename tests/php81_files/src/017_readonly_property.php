<?php
class Point {
    public function __construct(public readonly int $x, public readonly int $y) {}
}
$p = new Point(1, 2);
$p->x = 123;
echo spl_object_id($p->x);
var_dump($p->y);
