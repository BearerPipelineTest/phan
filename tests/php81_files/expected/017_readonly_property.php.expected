%s:7 PhanTypeMismatchArgumentInternalReal Argument 1 ($object) is $p->x of type int but \spl_object_id() takes object
